Dockerfile.no-systemd

```diff
@@ -1,4 +1,7 @@
-FROM balenalib/amd64-debian-node:18.12.1-bullseye-run
+# use TARGETARCH from buildkit to select the appropriate base image
+FROM balenalib/amd64-debian-node:18.12.1-bullseye-run as amd64
+FROM balenalib/aarch64-debian-node:18.12.1-bullseye-run as arm64
+FROM ${TARGETARCH}
```

Comment:
Can this be
```suggestion
FROM balenalib/${TARGETARCH}-debian-node:18.12.1-bullseye-run
```
?

Reply:
Unfortunately `TARGETARCH` and any architecture described in golang will be `arm64` and not `aarch64`.

Hence the need for the translation. Fortunately buildkit is smart and won't pull both base images, only the one it needs. I've done this in the past for a few projects to map golang architectures to balena architectures.

Reply:
The other obvious option is to use the official node base images, which I had originally, but introduces more change.

Reply:
Oh right, I didn't spot the aarch64 -> arm64 translation